### ISSUE TITLE
Editor: Links to user's site in draft posts display blank page in Preview

### ISSUE THREAD

**Issue by samjhamdy** [Automattic/wp-calypso#3220]
Ticket: 2514527-t
The use reported that clicking the Preview button for a draft post, then clicking a link within the post, leads to a blank page.

I've been able to replicate this on my test blog. Test draft in Calypso: https://wordpress.com/post/sjhtestdomain.com/3046

In testing this issue, I've discovered that only links to the user's site create this behavior. Links to external sites seem to work fine. I've labeled the various links I've tested in the above post in the link text for each link.

I've also created a link to the user's site on another test site that's not a replicate of the user's site: https://wordpress.com/post/sjhtest1.wordpress.com/4760 (Link text: 'Test of link in Preview')

The same behavior appears here - clicking the link in Preview leads to a blank page.

Steps to replicate:

1. Create a draft post in Calypso.
2. Add a link to a page or post on the site
3. Click the Preview button
4. Click on link in the preview

Screencast:

![calypso-preview-link](https://cloud.githubusercontent.com/assets/9888982/12957726/369ac68c-cffc-11e5-844b-2884ac7cc17d.gif)

Let me know if I can provide any additional information that would be helpful. Thanks!

**Comment by hoverduck** [comment]
21Apr16 Bug Scrub

I confirmed this is still a problem, but I have the opposite symptoms of the original report.  Links to pages internal to my site work fine, but links to external sites yield a blank page.  Regardless, once a link is clicked, there's no way to get back to the original post preview without an entire page refresh.

![3220-gh-wp-calypso](https://cloud.githubusercontent.com/assets/7233112/14718478/eb898060-07c4-11e6-88d7-cc28fa49c519.gif)

**Comment by designsimply** [comment]
Closing as I re-tested and was unable to reproduce the problem using the testing steps provided.

Tested using Firefox 47.0.1 on Mac OS X 10.11.5 on production. I found that clicking on a link to an external site from a draft preview resulted in the link opening in a new tab and clicking on a link to a post on the same blog opened the post. 

<blockquote>Regardless, once a link is clicked, there's no way to get back to the original post preview without an entire page refresh.</blockquote>

Raised this at https://github.com/Automattic/wp-calypso/issues/6911
